Product-page/product.page.html

```diff
@@ -157,7 +157,7 @@ <h3 class="price-now center">₹8,999</h3>
               alt="mens-jacket"
```

Comment:
if this not need , then remove this , might be broke your page

Reply:
ok